lib/simplecov/defaults.rb

```diff
@@ -29,6 +29,7 @@
   add_filter "/db/"
 
   add_group "Controllers", "app/controllers"
+  add_group "Channels", "app/channels"
```

Comment:
could be something like  `  add_group "Channels", "app/channels" if defined?(ActionCable) || defined?(Rails) && Rails::VERSION::MAJOR >= 5` but the below might be cleaner.   

```ruby
SimpleCov.profiles.define "rails5" do
  load_profile "rails"

  add_group "Channels", "app/channels"
end
```
 
@colszowka @xaviershay Any preference for adding the group here in all Rails version, with a condition, or making a new rails5 profile?